src/Router.jsx

```diff
@@ -1,17 +1,23 @@
 import { BrowserRouter, Route, Routes, Navigate } from 'react-router-dom';
```

Comment:
Chat is not working 
![image](https://user-images.githubusercontent.com/85763715/152655203-6882f978-6ff4-4c22-9dad-7c237bcf19b1.png)

Reply:
ok i see chat page is returning null but why?

Reply:
temporary, waiting for its own pull request

Reply:
ok, but I think that what was there might stay, because UI was done, all to do is just connet this with api